matter/efr32/efr32mg24/BRD2601B/config/sl_power_manager_config.h

```diff
@@ -57,6 +57,20 @@
 #define SL_POWER_MANAGER_DEBUG_POOL_SIZE  10
 // </e>
 
+// <o SL_POWER_MANAGER_INIT_EMU_EM4_PIN_RETENTION_MODE> Pin retention mode
+// <i>
+// <EMU_EM4CTRL_EM4IORETMODE_DISABLE=> No retention
+// <EMU_EM4CTRL_EM4IORETMODE_EM4EXIT=> Retention through EM4
+// <EMU_EM4CTRL_EM4IORETMODE_SWUNLATCH=> Retention through EM4 and wakeup
+// <d> power_manager_pin_retention_disable
+#define SL_POWER_MANAGER_INIT_EMU_EM4_PIN_RETENTION_MODE EMU_EM4CTRL_EM4IORETMODE_DISABLE
+
+// <e SL_POWER_MANAGER_INIT_EMU_EM2_DEBUG_ENABLE > Enable EM2 debugging feature
+// <i> Enable or disable debugging features.
+// <i> Default: 1
+#define SL_POWER_MANAGER_INIT_EMU_EM2_DEBUG_ENABLE 1
```

Comment:
Do we want to keep this debug define?

Reply:
I don't know what it enables. I'll search a bit. But probably we can set to 0

Reply:
It only sets a register bit to 1 when enabled. I think we can let it set if it allows us to debug em2 by default